website/content/docs/upgrading/instructions/index.mdx

```diff
@@ -28,6 +28,9 @@ The upgrade guides will mention notable changes and link to relevant changelogs
 we recommend reviewing the changelog for versions between the one you are on and the
 one you are upgrading to at each step to familiarize yourself with changes.
 
+~> **Note:** If you are upgrading from `1.11`+ and have connect proxies 
```

Comment:
In the "Upgrade Path" section, we list the path:
```
...
- 1.8.19 (final 1.8.x)
- Latest 1.10.x
- Latest current version (1.11.x or 1.12.x)
```

... which currently makes no mention of 1.13. With recent major versions (e.g., 1.6+), we don't want users to jump more than 2 major versions at a time (which is something we should document more explicitly in a separate PR).

Perhaps we should remove these new lines here (since there's nothing for 1.10+ yet). And instead, make the following changes to the table above...

---

```
Our recommended upgrade path is to move through the following sequence of versions:
- 0.8.5 (final 0.8.x)
- 1.2.4 (final 1.2.x)
- 1.6.10 (final 1.6.x)
- 1.8.19 (final 1.8.x)
- Latest 1.10.x
- Latest 1,12,x
- Latest 1.13.x ([at least 1.13.1](#/docs/upgrading/upgrade-specific#service-mesh-compatibility))
```

Reply:
The link `/docs/upgrading/upgrade-specific#service-mesh-compatibility` is in reference to my proposed change in [my other comment](https://github.com/hashicorp/consul/pull/14154/files#r943933232)